From 2dd87a3f4ef2346a74f61e8239b8ba5eeb7e6a6f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Jirman?= <megi@xff.cz>
Date: Sun, 21 Nov 2021 17:02:50 +0100
Subject: [PATCH 177/469] arm64: dts: rk3399-pinebook-pro: Remove unused
 features

I don't use PCIe on Pinebook Pro and having it enabled just produces
probe failures and delays.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
index b910157a5f4d..1b6f78bdb08c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinebook-pro.dts
@@ -441,10 +441,6 @@ &gpu {
 	status = "okay";
 };
 
-&hdmi_sound {
-	status = "okay";
-};
-
 &i2c0 {
 	clock-frequency = <400000>;
 	i2c-scl-falling-time-ns = <4>;
@@ -798,7 +794,7 @@ &io_domains {
 };
 
 &pcie_phy {
-	status = "okay";
+	//status = "okay";
 };
 
 &pcie0 {
@@ -810,7 +806,7 @@ &pcie0 {
 	vpcie0v9-supply = <&vcca0v9_s3>;
 	vpcie1v8-supply = <&vcca1v8_s3>;
 	vpcie3v3-supply = <&vcc3v3_ssd>;
-	status = "okay";
+	//status = "okay";
 };
 
 &pinctrl {
-- 
2.34.1

